_posts/2019-04-06-omakase-relay.markdown

```diff
@@ -0,0 +1,200 @@
+---
+layout: epic
+title: "Why does Artsy use Relay?"
+date: "2019-04-06"
+author: [orta]
+categories: [community, omakase, relay]
+---
+
+When the mobile team at Artsy considered moving to React Native back in 2014, one of the most compelling cases for
+making that jump was Relay. This, it seems, is a dependency that is rarely used in the JS community and we often
+find ourselves defending this decision to new engineers during onboarding. It's great to [..]
+
+Let's have a deep dive into what makes Relay compelling for Artsy Engineering.
+
+<!-- more -->
+
+# What problem does Relay solve?
+
+Relay is an API client for GraphQL, it comes in two parts: a compiler and a set of front-end components. Relay aims
+to provide a really tight binding between your GraphQL API and your view hierarchy. When you build data-driven
+apps, Relay removes a whole suite of non-business logic from your application.
+
+Relay handles:
+
+- Data binding (API -> props)
+- Data transformations (response shaping)
```

Comment:
What do you mean by this exactly?

Reply:
Good point. I think it doesn't add enough value to be in there. But my original intention was to encompass local schema extensions (without the direct jargon) but we don't use it and it's an arcane API ( https://github.com/facebook/relay/issues/1787 ) so I'm killing it